example/ios/VisionCameraExample/Info.plist

```diff
@@ -35,8 +35,19 @@
 			</dict>
 		</dict>
 	</dict>
+	<key>NSCameraUsageDescription</key>
+	<string>Allow Ocula App access to the Camera?</string>
 	<key>NSLocationWhenInUseUsageDescription</key>
 	<string></string>
+	<key>NSMicrophoneUsageDescription</key>
+  <string>Allow Ocula App access to the Microphone?</string>
+	<key>NSPhotoLibraryUsageDescription</key>
+  <string>Allow Ocula App to save data to the Photo Library?</string>
+	<key>UIAppFonts</key>
+	<array>
+		<string>Ionicons.ttf</string>
+		<string>MaterialCommunityIcons.ttf</string>
+	</array>
```

Comment:
Still need to remove this so we can create the PR against the library itself once we're merged.  Maybe keep it in an alternate branch?